cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -289,10 +289,8 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
 
         log.perf { "Executed the evaluator in ${duration.inWholeMilliseconds}ms." }
 
-        if (log.delegate.isErrorEnabled) {
-            evaluatorRun.violations.forEach { violation ->
-                log.error { violation.format() }
-            }
+        evaluatorRun.violations.forEach { violation ->
+            log.log(violation.severity.toLog4jLevel()) { violation.format() }
```

Comment:
I believe previous `error` log level didn't make sense, because I would regard a violation an error. 
It's just the output of a successfuly evaluator run. From that perspective I believe the same log level should be used for all violations, as otherwise one can easily get the false impression of viewing the whole summary instead of just the filtered one. 

What do you think about using log.info for all violations?